scandium-core/src/main/java/org/eclipse/californium/scandium/dtls/CertificateRequest.java

```diff
@@ -509,9 +509,15 @@ public SignatureAndHashAlgorithm getSignatureAndHashAlgorithm(PublicKey key) {
 	 *         compatible with any of the supported certificate types or any of the supported signature algorithms.
 	 */
 	public SignatureAndHashAlgorithm getSignatureAndHashAlgorithm(List<X509Certificate> chain) {
```

Comment:
This method sounds never used to me ? 

Should it be a list returned ? I mean for a certChain it could be several algorithm ?

Reply:
Typo in `ClientHandshaker.createCertificateMessage`. Now it's used again.